Dfe.Data.SearchPrototype/Web/Mappers/EstablishmentFacetsToFacetsViewModelMapper.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Dfe.Data.SearchPrototype.Common.Mappers;
+using Dfe.Data.SearchPrototype.SearchForEstablishments.Models;
+using Dfe.Data.SearchPrototype.Web.ViewModels;
+
+namespace Dfe.Data.SearchPrototype.Web.Mappers
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public sealed class EstablishmentFacetsToFacetsViewModelMapper : IMapper<EstablishmentFacetsMapperRequest, List<Facet>?>
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="input">
+        /// 
+        /// </param>
+        /// <returns>
+        /// 
+        /// </returns>
+        public List<Facet>? MapFrom(EstablishmentFacetsMapperRequest input)
```

Comment:
I don't really like this EstablishmentFacetsMapperRequest - maybe it's just the name I don't like - or could we just send a Tuple 😱

Reply:
rename instead of request

Reply:
Rename completed.